### PR TITLE
Governance docs reorg

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: DOCS-649

# Sanity check

The sole purpose of OP inflation is to have funds for RPGF (README.md)

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -116,17 +116,80 @@ module.exports = {
...
+            },     
+            {
+              title: "Delegates",
+              collapsable: true,
+              children: [
+                [
+                  "https://app.optimism.io/delegates",
+                  "Delegate your tokens' voting power"
+                ],
+                '/docs/governance/delegate.md',
```
Comment (Nicca42) [inline src/.vuepress/config.js:155]: Can we move this to being the first page please

File: src/docs/governance/delegate.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: Being a Token House delegate
```
Comment (Nicca42) [inline src/docs/governance/delegate.md:2]: This should be "Become a Token House delegate

File: src/docs/governance/gov-fund.md [hunk 3]
```diff
@@ -1,18 +1,11 @@
 ---
-title: Governance Fund Overview
+title: Governance fund overview
```
Comment (Nicca42) [inline src/docs/governance/gov-fund.md:2]: Make caps

File: src/docs/governance/token-house-history.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: Token House history
```
Comment (Nicca42) [inline src/docs/governance/token-house-history.md:2]: Caps

File: src/docs/governance/token-house.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+title: Token House governance
```
Comment (Nicca42) [inline src/docs/governance/token-house.md:2]: Caps

File: src/docs/governance/proposals.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: Submitting proposals
```
Comment (Nicca42) [inline src/docs/governance/proposals.md:2]: Caps

File: src/docs/governance/delegate.md (hunk 2 above)
Comment (Nicca42) [reply src/docs/governance/delegate.md:2]: Also Caps

File: src/docs/governance/citizens-house.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+title: Citizens' House governance
```
Comment (Nicca42) [inline src/docs/governance/citizens-house.md:2]: Caps

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/governance/delegate.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/governance/delegate.md:2]: "Become" implies this doesn't include the information for existing delegates, which is not true. How about "Information for Delegates"?

File: src/docs/governance/token-house-history.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+### Season #2: Sep 8th - Nov 9th, 2022
+
+<!--
+* Voting cycle #8 (Oct 20th - Nov 9th)
+* Voting cycle #7 (Sep 29th - Oct 19th)
+-->
+* Voting cycle #6 (Sep 8th - Sep 28th)
+
+### Voting Cycle #5
```
Comment (Nicca42) [inline src/docs/governance/token-house-history.md:52]: Please make this a subheading of season 2 (cycle 5) 
reflection period can stay as a high level.

File: src/docs/governance/token-house-history.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+Changes made during the Reflection Period are voted on during two-week Special Voting Cycles.
+
+### Reflection Period #1 
+
```
Comment (Nicca42) [inline src/docs/governance/token-house-history.md:57]: Add a one-liner here: 

The reflection period was a break between season 1 and season 2 where the Optimism foundation, with the help of delegates and community members, updated and improved the governance process.

File: src/docs/governance/proposals.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
+---
+title: Submitting Proposals
+lang: en-US
+---
+
+## Grant Proposals
+
```
Comment (Nicca42) [inline src/docs/governance/proposals.md:7]: Please add a tip here: 

This info is here for your convenience. The source of truth for all governance related processes is the OPerating Manual (link).

File: src/docs/governance/howto-delegate.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+Would you rather vote yourself? 
+No problem, but you’ll have to delegate to yourself first! 
+If you try to vote with your tokens but have not delegated to yourself, you will not be able to vote. 
+
+1. [Delegate to yourself](https://help.optimism.io/hc/en-us/articles/6296720540955-How-do-I-delegate-to-myself-).
+
+1. [Go to the delegates app](https://app.optimism.io/delegates) to see that you’ve delegated your tokens to your own address.
+
+   Please note that delegation only becomes effective the next time [Snapshot](https://snapshot.org/#/opcollective.eth) measures how many votes each delegate has. 
+   This happens at the start of every voting cycle, which is every three weeks.
```
Comment (Nicca42) [inline src/docs/governance/howto-delegate.md:48]: This happens at the start of the voting phase in a cycle, which is the last week of each cycle. A new cycle starts every 3 weeks.

File: src/docs/governance/gov-fund.md [hunk 12]
```diff
@@ -3,104 +3,43 @@ title: Governance Fund Overview
...
 Projects are also encouraged to reference data from the success of their initial distribution to strengthen their case in future proposals.
 
-### Matching
 
-Although not a strict requirement, the Optimism Foundation recommends that each project submitting a Governance Fund proposal include a commitment to match OP token incentives with their own project incentives (when the project has the capital and capacity to do so).
+## Phase 0
+
+Phase 0 was designed to reward existing projects that have driven growth of the Optimism ecosystem. 
+
+The amount of funding for each project was determined by a set of on-chain metrics, calculated via a snapshot of the Optimism chain on April 1, 2022 at 12:00am UTC.
```
Comment (Nicca42) [inline src/docs/governance/gov-fund.md:45]: To see live tracking on govFund allocations please see the [Public tracker spreadsheet](https://docs.google.com/spreadsheets/d/1eaHOlWB34ij1KGsXdaNyTQg4tt1Pu2JurJsElBb6a8k/edit#gid=0)

File: src/docs/governance/howto-delegate.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+   Your delegate should also write up the reasoning behind their votes in a [Delegate Communications Thread](https://gov.optimism.io/c/governance/41).
+
+Delegation is always in your control. You can return to the [delegation page](https://app.optimism.io/delegates) at any time to change your delegate selection, or to re-delegate voting power to yourself (more below).
+
+### How do I delegate to myself?
+
+Would you rather vote yourself? 
+No problem, but you’ll have to delegate to yourself first! 
+If you try to vote with your tokens but have not delegated to yourself, you will not be able to vote. 
+
```
Comment (Nicca42) [inline src/docs/governance/howto-delegate.md:42]: I would just link to the section https://github.com/ethereum-optimism/community-hub/pull/473/files#diff-f1c2180dcfc5e0112202d868c037754121d35d92be8cdc0b8e9d04599000527fR18 here where you talk about how to commit. Something like

if you want to become a delegate that others delegate to you need to [state your delegation commitment]()

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/governance/token-house-history.md (hunk 8 above)
Comment (qbzzt) [reply src/docs/governance/token-house-history.md:52]: Shouldn't it be if anything under **Reflection Point #1**?

File: src/docs/governance/howto-delegate.md (hunk 13 above)
Comment (qbzzt) [reply src/docs/governance/howto-delegate.md:42]: Trevor asked for this change, so I am just adding your sentence.

File: src/docs/governance/token-house-history.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+## History
+
+<!--
+### Season #3: Dec 15th, 2022 - Feb 15th, 2023
+
+* Voting system #11 (Jan 26th - Feb 15th)
+* Voting system #10 (Jan 5th - Jan 25th)
+* Voting system #9 (Dec 15th, 2022 - Jan 4th, 2023)
+
+### Special Voting Cycle #2: Dec 1st - Dec 14th, 2022
```
Comment (Nicca42) [inline src/docs/governance/token-house-history.md:38]: This needs to be cycle 9 (its messed up your numbers)

File: src/docs/governance/token-house-history.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+:::
+
+## History
+
+<!--
+### Season #3: Dec 15th, 2022 - Feb 15th, 2023
+
+* Voting system #11 (Jan 26th - Feb 15th)
+* Voting system #10 (Jan 5th - Jan 25th)
+* Voting system #9 (Dec 15th, 2022 - Jan 4th, 2023)
```
Comment (Nicca42) [inline src/docs/governance/token-house-history.md:36]: Then this is cycle 10

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/governance/token-house-history.md (hunk 8 above)
Comment (Nicca42) [reply src/docs/governance/token-house-history.md]: hm. No I think we specified it as the first voting cycle of season two. It is to ratify the learnings from the reflection period.

Comment (qbzzt) [reply src/docs/governance/token-house-history.md]: Then each season starts two weeks earlier. I'll update that.

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**